dist/fabric.js

```diff
@@ -20530,14 +20533,14 @@ fabric.util.object.extend(fabric.Object.prototype, /** @lends fabric.Object.prot
       var newGroup = new fabric.Group([]);
       delete options.type;
       newGroup.set(options);
-      var firstIndex = null
+      var firstIndex = null;
       objects.forEach(function(object) {
-        // We record the *lowest* index of object in the group. We can't use 
+        // We record the *lowest* index of object in the group. We can't use
         // the highest index because in the case of dependent objects, removing
         // one from the collection might also trigger removal of others
         // changing all of the higher indexes.
         var objectIndex = object.canvas.indexOf(object);
-        if (!firstIndex || objectIndex < firstIndex) firstIndex = objectIndex;
+        if (!firstIndex || objectIndex < firstIndex) {firstIndex = objectIndex;}
```

Comment:
Just a stylistic nit, not a blocker. If we add braces, it's easier (imo) to break this into separate lines like:

```suggestion
        if (!firstIndex || objectIndex < firstIndex) {
          firstIndex = objectIndex;
        }
```